ts/Series/HLC/HLCSeries.ts

```diff
@@ -0,0 +1,485 @@
+                path: SVGPath,
+                graphic = point.graphic,
+                crispX,
+                isNew = !graphic,
+                strokeWidth;
+
+            if (typeof point.plotY !== 'undefined') {
+
+                // Create and/or update the graphic
+                if (!graphic) {
+                    point.graphic = graphic = chart.renderer.path()
+                        .add(series.group);
+                }
+
+                if (!chart.styledMode) {
+                    graphic.attr(
+                        series.pointAttribs(
+                            point,
+                            (point.selected && 'select') as any
+                        )
+                    ); // #3897
+                }
+
+                // crisp vector coordinates
+                strokeWidth = graphic.strokeWidth();
+                crispCorr = (strokeWidth % 2) / 2;
+                // #2596:
+                crispX = Math.round(point.plotX as any) - crispCorr;
+                halfWidth = Math.round((point.shapeArgs as any).width / 2);
+
+                // the vertical stem
+                path = [
+                    ['M', crispX, Math.round(point.yBottom as any)],
+                    ['L', crispX, Math.round(point.plotHigh as any)]
+                ];
+
+                // close
+                if (point.close !== null) {
+                    plotClose = Math.round(point.plotClose) + crispCorr;
+                    path.push(
+                        ['M', crispX, plotClose],
+                        ['L', crispX + halfWidth * 2, plotClose]
+                    );
+
+                    extendStem(path, strokeWidth / 2, plotClose);
+                }
+
+                graphic[isNew ? 'attr' : 'animate']({ d: path })
+                    .addClass(point.getClassName(), true);
+
+            }
+
+
+        });
+
+    }
+
+    /**
+     * @private
+     * @function Highcarts.seriesTypes.hlc#init
+     * @return {void}
+     */
+    public init(): void {
+        super.init.apply(this, arguments as any);
+
+        this.options.stacking = void 0; // #8817
+    }
+
+    /**
+     * Postprocess mapping between options and SVG attributes
+     * @private
+     */
+    public pointAttribs(
+        point: HLCPoint,
+        state: StatesOptionsKey
+    ): SVGAttributes {
+        const attribs = super.pointAttribs.call(
+                this,
+                point,
+                state
+            ),
+            options = this.options;
+
+        delete attribs.fill;
+
+
+        return attribs;
+    }
+
+    public toYData(point: HLCPoint): Array<number> {
+        // return a plain array for speedy calculation
+        return [point.high, point.low, point.close];
+    }
+
+    /**
+     * Translate data points from raw values x and y to plotX and plotY
+     *
+     * @private
+     * @function Highcharts.seriesTypes.hlc#translate
+     * @return {void}
+     */
+    public translate(): void {
```

Comment:
`translate` for `HLC` and `OHLC` series are almost identical. We need to optimize this

Reply:
`init` and `drawPoints` too. Please check all methods, since the HLC series will most likely get into the core